scripts/provisioning/vmhost/gci-create-cluster.sh

```diff
@@ -256,7 +256,7 @@ clone_hare()
     fi
 
     print_msg "Executing [git merge --no-ff $USER/hare-$HARE_USER_BRANCH]"
-    git merge --no-ff $USER/hare-$HARE_USER_BRANCH
+    git merge --no-ff "$USER"/hare-"$HARE_USER_BRANCH"
```

Comment:
It's better to have:
 git merge --no-ff "$USER/hare-$HARE_USER_BRANCH"

Reply:
done